src/scss/06_components/paragraphs/_event-list.scss

```diff
@@ -8,186 +8,122 @@ $event-list-image-mobile-height: 264px;
 $tag-horizontal-padding: 12px;
 $tag-vertical-padding: 5px;
 
-.event-list__event {
-  border-bottom: 1px solid $color-black-10;
-  padding-bottom: $spacing-half;
-  padding-top: $spacing-and-half;
-  position: relative;
-
-  @include breakpoint($breakpoint-l) {
-    display: grid;
-    grid-template-columns: $event-list-image-width 1fr;
-    padding-bottom: $spacing;
-    padding-top: $spacing;
-  }
+.component--event-list {
+  background-color: $color-silver-light;
+}
 
-  &:first-child {
-    border-top: 1px solid $color-black-10;
-  }
+// Since the component is not "full width" next to a sidebar, we should uniform padding around
+.has-sidebar .components--upper .component--event-list .component__container {
+  padding-bottom: $spacing-double;
+  padding-left: $spacing-double;
+  padding-right: $spacing-double;
+  padding-top: $spacing-double;
```

Comment:
If you need to define all paddings you can use the shorthand `padding: $spacing-double;`